src/dso_api/dynamic_api/datasets.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+from django.db.models import Q
+from schematools.contrib.django.models import Dataset
+
+
+def get_published_datasets(queryset=Dataset.objects):
```

Comment:
better have `base_queryset=None` and then call `Dataset.objects` inside the method. Now it's a compile-time call.

Reply:
ja ik bedacht dat 'active' eigenlijk dezelfde ambiguiteit heeft als 'published', maar published is wel descriptiever.

Reply:
Hmm goed punt.. Ik vind "active" nog wel duidelijk: welke je geactiveerd hebt. Want er zijn veel meer datasets publiceert.

Reply:
Ahh de term 'published' wordt al gebruikt? Dan verander ik hem